GuysNight.LethalCompanyMod.BalancedItems/Patches/RoundManagerPatches.cs

```diff
@@ -27,13 +27,7 @@ public class RoundManagerPatches {
 					return;
```

Comment:
This should be a continue for safety. Overrides should exist for all items at this point, but mind as well make this continue just in case.